xla/service/gpu/gpu_hlo_schedule.cc

```diff
@@ -332,6 +332,22 @@ class GpuAsyncTrackerBase : public AsyncTracker {
             !IsSyncCollective(&hlo)) ||
            IsAsyncComputeOp(hlo);
   }
+
+  void PostProcessScheduleGraph(
+      HloScheduleGraph* schedule_graph,
+      const LatencyEstimator* latency_estimator) const override {
+    for (auto inst : schedule_graph->GetOriginalInstrList()) {
+      if (inst->has_backend_config()) {
+        auto gpu_config = inst->backend_config<GpuBackendConfig>();
+        if (gpu_config.ok()) {
```

Comment:
and it it isn't .ok() ? Are we fine with instructions having or not having GpuBackendConfig?

Reply:
if the instruction doenst have gpu config, then we dont need to look at that, we will simply skip to the next instruction

Reply:
What instructions don't have a GPU config? I thought they all do?

Reply:
There are a few that dont have a gpu config, i.e `while` or some jax custom calls that only have backend_config as an encoded string.